Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -25,14 +25,15 @@ public protocol ProtobufBinaryCodableType: ProtobufTypePropertiesBase {
 
     static func setFromProtobufVarint(varint: UInt64, value: inout BaseType?) throws -> Bool
     static func setFromProtobufVarint(varint: UInt64, value: inout [BaseType]) throws -> Bool
```

Comment:
Hard to tell skimming the diffs, why do some still return Bools and others throw?